### DISCUSSION TITLE
Note of appreciation

### DISCUSSION THREAD

**Discussion by IlywI** [Monika-After-Story/MonikaModDev#8778] [appreciation]
_Writing this since I felt that this was long overdue. And I posted this in issue as I do not know of any other way to get this through to the contributors so i really hope that all of you can see this._

I'm really grateful that this mod is still getting updated. From the looks of it, there exists only a handful of contributors out there that are still pushing commits to the repository. No words can describe my appreciation for the contributors contributing their time from their everyday lives to work on this mod perhaps not knowing the number of people who even use the mod nowadays. So all i can really say is thank you all for your hard work and keep up the good work.

**Comment by Rai99** [comment]
_Every day, I imagine a future where_ contributorPRsactuallygetreviewedandmergedinatimelymanner. 

**18** open PRs ready and waiting for review from me and still going strong. If I get to 25 open PRs or enough to fill my own page of PRs, I might just ask to become a dialogue reviewer myself. 

I'm glad people supposedly like the work I put in even though you might be talking about other contributors, and I plan to keep churning out PRs from time to time.

**Comment by multimokia** [comment]
Let's not forget artists! The art in the game is great and without it, mas wouldn't be nearly what it is today, and of course the music too! Where a few artists have also put their efforts into making new music for the mod

Nice to see people's work appreciated

**Comment by Rai99** [comment]
^^ #1897 #1901 ^^

Don't forget the devs and everyone on the team for making all of this possible. ~~Even though contributor PRs are seriously backed up.~~ They also deserve appreciation for their work making sure MAS keeps evolving.

**Comment by loverofmonika** [comment]
i love this mod and its an honor to contribute even though i just give suggestions and some dialogue writing
